en/editor/project/index.md

```diff
@@ -0,0 +1,86 @@
+# Project Setting
+
+The Project Settings windows are available from Cocos Creator 3D’s main menu (Project > Project Setting) which includes all the settings related to your project. These settings will be saved in the project's `settings / packages` folder. If you need to synchronize project settings between different developers, this folder should be added in your source control system.
+
+## General
+
+![general](./index/general.jpg)
+
+### Default Canvas settings
+
+The default Canvas settings include design resolution and adapted screen width/height, which are used to specify the default design resolution value in Canvas when creating a new scene or Canvas component, as well as the `Fit Height, Fit Width` options.
+
+For more information, please refer to [Multi-resolution adaptation scheme](../ui-system/components/engine/multi-resolution.md).
+
+## Engine Modules
+
+![modules](./index/modules.jpg)
+
+The setting here is to crop the modules used in the engine to reduce the size of the released engine. Modules not selected in the panel will be cropped when they are packaged and previewed.It is recommended to perform a complete test after released to avoid using cropped modules in scenes and scripts.
+
+## Macro Config
+
+For more information and code of the engine macro module, please refer to [Engine macro](https://github.com/cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824).
+
+This panel here provide the convenience to modify the macro configuration. The **Macro Config** will take effect during preview and build. At the same time, the default value of the current macro configuration will be updated with the configuration of the custom engine.
+
+![macro](./index/macro.png)
+
+## Texture Compress
+
+> Since v1.2, the editor has modified the use of compressed texture configuration to configure presets in project settings and select presets for image asset's inspector. After the old version of the project is upgraded, the editor will automatically scan all the compressed texture configurations in the project and sort out as several presets.
+
+Used to add compressed texture preset configuration, you can directly select the compressed texture preset to quickly add in the inspector of image asset. At the same time, after adding presets, you can also directly modify the presets to update batch texture compress configuration. Project settings allow users to add multiple compressed texture configurations, and each compressed texture configuration allows to add different format for different platform categories.
+
+Platform is rough devised as following:
+
+1. Web: refers to the two platforms Web-Mobile and Web-Desktop
+2. Mac & Windows
+3. iOS
+4. Mini Game: Refers to the mini-games of various manufacturers' platforms, such as WeChat Mini Games and Huawei Quick Game Waiting;
+5. Android
+
+For the detail of the texture compression support of the platforms, please refer to [Compressed Texture Chapter](../../asset/compress-texture.md).
```

Comment:
For the detail of the texture compression support of the platforms, please refer to [Compressed Texture Chapter](../../asset/compress-texture.md).
-->
For the detail of the texture compression support of the platforms, please refer to the [Compressed Texture Chapter](../../asset/compress-texture.md) documentation.

Reply:
You missed "the"
--> please refer to the